### PR TITLE
Fix some opposite predicates should be inverses

### PR DESCRIPTION
**Pull request by vincentvialard**

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -1678,8 +1678,6 @@ slots:
       - OMIM:has_manifestation
       - SNOMED:has_basic_dose_form
       - UMLS:RB
-    annotations:
-      opposite_of: subclass of
```
Comment (sierra-moxon) [inline biolink-model.yaml:1682]: I think `subclass of` is the opposite of `superclass of` -- the inverse here would be `has superclass` but I do see the issue here, where we declare it both the inverse and the opposite.  Will fix.

Comment (balhoff) [reply biolink-model.yaml:1682]: Isn't `has superclass` just another way to say `subclass of`? I agree with @vincentvialard that this is an `inverse` situation, not opposite.

Comment (sierra-moxon) [reply biolink-model.yaml:1682]: I think `has superclass` and `superclass of` are basically saying the same thing but in inverse directions.

Comment (balhoff) [reply biolink-model.yaml:1682]: Yes but `has superclass` and `subclass of` are just synonyms of each other.

Comment (sierra-moxon) [reply biolink-model.yaml:1682]: practically, I agree with you; in biolink, they are two distinct predicates?

Comment (balhoff) [reply biolink-model.yaml:1682]: I don't see `has superclass` in the model; I think I'm just introducing confusion here. :-)

Comment (sierra-moxon) [reply biolink-model.yaml:1682]: after talking a little more with Jim and the Biolink team today, we'll remove the opposite designation from subclass_of and superclass_of, as well as the inverse.

**Pull request merged**